Commander/ConnectedCommands.cs

```diff
@@ -66,6 +66,14 @@ public ConnectedContext(AuthCommon auth)
                         });
                 }
 
+                Commands.Add("get",
```

Comment:
What was the reason of moving `get` command from Vault to Connected?
This command seems to use only vault functionality.